main.py

```diff
@@ -62,7 +64,11 @@ def buy():
                 announcement_coin not in order and \
                 announcement_coin not in sold_coins and \
                 announcement_coin not in globals.old_coins:
-            logger.info(f'New announcement detected: {announcement_coin}')
+            
+            message = f'New announcement detected: {announcement_coin}'
+            logger.info(message)
+            send_telegram(message, 'COIN_ANNOUNCEMENT')
```

Comment:
I feel like this flow could be improved. Instead of building the `message` variable, then sending the message to both the logger and the `send_telegram` function, Telegram functionality should be added to `logger.py`. When a logging message is called, we can add the telegram category, maybe as follows:
```python
logger.info("Placing buy order", 'BUY_START')
```

When logger is called, we can check to see if there is a telegram category with the message, and if there is, we send the telegram message. A similar concept is used in [PR 48](https://github.com/CyberPunkMetalHead/gateio-crypto-trading-bot-binance-announcements-new-coins/pull/48). This will also help in the future with new functionality added so any contributors don't have to worry about logging AND adding telegram notifications.

You can also see this discussion in discord: https://discord.com/channels/900656563805642753/900735456449290321/912884981020360716

Reply:
Sounds good to me, I will change accordingly

Reply:
@HebelHuber Thanks for all the hard work on this. The feature has been requested many times and I just want to make sure it works well before we merge.

Reply:
Sure, enjoy working on this.
Also my first PR, so it better be good :)